opentelemetry/chart/templates/metrics-collector.yaml

```
@@ -3,6 +3,7 @@ SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse c
 SPDX-License-Identifier: Apache-2.0
 */}}
 
+{{- if index .Values "open_telemetry" "MetricsCollector" "enabled" }}
```

Same here. The index function is not needed. 
```suggestion
{{- if .Values.openTelemetry.MetricsCollector.enabled }}
```